server/database/init.sql

```diff
@@ -96,21 +97,21 @@ VALUES
     ('0d76041e-54ce-4cea-a128-ebfa32171c29', 'PNI:93c5486c-5bba-437f-a9c1-0570cb619d27', '\x0531840a62513f5a0388b658978aba393e1501cc3438588019fbd8d6136f17f815'::BYTEA, '\x05c288f7a6bf5321cf151b596c7ea9cb1d279576affce0d671992baba7f19b76e7'::BYTEA, '12345678', '\x00010000000100000000000000000000000000000000'::BYTEA); -- keys have to start with 05 and be over 64 bytes
 
 INSERT INTO 
-    devices (owner, device_id, name, auth_token, salt, registration_id)
+    devices (owner, device_id, name, auth_token, salt, registration_id, pni_registration_id)
 VALUES
-    (1, '1', '\x1234'::BYTEA , '\x2998fa253a711f6585de13634b50e4b5c67646307cb903b8c3e6febba813937d'::BYTEA, 'salt', '0'); -- password = password 
+    (1, '1', '\x1234'::BYTEA , '\x2998fa253a711f6585de13634b50e4b5c67646307cb903b8c3e6febba813937d'::BYTEA, 'salt', '0', '0'); -- password = password 
 
 INSERT INTO 
-    devices (owner, device_id, name, auth_token, salt, registration_id)
+    devices (owner, device_id, name, auth_token, salt, registration_id, pni_registration_id)
```

Comment:
Should the other registrations_id be aci_registration_id then?

Reply:
No